breathecode/events/views.py

```diff
@@ -379,6 +379,19 @@ def delete(self, request, format=None, academy_id=None):
         return Response(None, status=status.HTTP_204_NO_CONTENT)
 
 
+# list eventbride webhook
+class OrganizationWebhookView(APIView):
+    @capable_of('read_organization')
+    def get(self, request, academy_id=None):
+
+        webhooks = EventbriteWebhook.objects.filter(academy__id=academy_id)
```

Comment:
How did you filter by academy if there EventbriteWebhook model does not have an academy property?
https://github.com/breatheco-de/apiv2/blob/master/breathecode/events/models.py#L224